src/extensions/default/brackets-show-whitespace/main.js

```diff
@@ -0,0 +1,319 @@
+/*
+ * The MIT License (MIT)
+ * Copyright (c) 2012-2015 Dennis Kehrig. All rights reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the "Software"),
+ * to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense,
+ * and/or sell copies of the Software, and to permit persons to whom the
+ * Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ *
+ */
+
+/*jslint vars: true, plusplus: true, devel: true, regexp: true, nomen: true, indent: 4, maxerr: 50 */
+/*global define, $, brackets */
+
+define(function (require, exports, module) {
+    "use strict";
+    
+    
+    // --- Required modules ---
+
+    var _                  = brackets.getModule("thirdparty/lodash"),
+        AppInit            = brackets.getModule("utils/AppInit"),
+        CommandManager     = brackets.getModule("command/CommandManager"),
+        EditorManager      = brackets.getModule("editor/EditorManager"),
+        ExtensionUtils     = brackets.getModule("utils/ExtensionUtils"),
+        Menus              = brackets.getModule("command/Menus"),
+        PreferencesManager = brackets.getModule("preferences/PreferencesManager"),
+        stylesTemplate     = require("text!styles/whitespace-colors-css.tmpl"),
+        Strings            = require("strings");
+
+    
+    // --- Settings ---
+    
+    var commandID          = "denniskehrig.ShowWhitespace.toggle";
+    var preferencesID      = "denniskehrig.ShowWhitespace";
+    var defaultPreferences = {
+        enabled: true,
```

Comment:
Let's change this to `false` by default, so that we have to flip a pref in order to enable it.  Then you can load it all the time.